modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -4867,25 +4866,25 @@ private void _validateListTypeEntryKey(
 	}
 
 	private void _validateObjectStateTransition(
-			Map.Entry<String, Serializable> entry, long listTypeDefinitionId,
-			ObjectEntry objectEntry, long objectFieldId, long userId)
+			ObjectEntry existingObjectEntry, long listTypeDefinitionId,
+			ObjectField objectField, long userId, Serializable value)
 		throws PortalException {
 
-		Map<String, Serializable> values = objectEntry.getValues();
+		Map<String, Serializable> values = existingObjectEntry.getValues();
 
-		Serializable value = values.get(entry.getKey());
+		Serializable existingValue = values.get(objectField.getName());
```

Comment:
We should choose between: `values / value` OR `existingValues / existingValue`.
I would choose: `existingValues / existingValue`.

Reply:
@marcelabc Done!